scripts/test-check-packages.sh

```diff
@@ -67,7 +67,15 @@ for d in test/packages/${PACKAGE_TEST_TYPE:-other}/${PACKAGE_UNDER_TEST:-*}/; do
     elastic-package install -v
 
     if [ "${PACKAGE_TEST_TYPE:-other}" == "benchmarks" ]; then
+      rm -rf "${OLDPWD}/build/benchmark-results"
+      elastic-package benchmark -v --report-format xml --report-output file --fail-on-missing
+      
+      rm -rf "${OLDPWD}/build/benchmark-results-old"
+      mv "${OLDPWD}/build/benchmark-results" "${OLDPWD}/build/benchmark-results-old"
+      
       elastic-package benchmark -v --report-format json --report-output file --fail-on-missing
+      
+      elastic-package report --fail-on-missing benchmark --new ${OLDPWD}/build/benchmark-results --old ${OLDPWD}/build/benchmark-results-old --threshold 1
```

Comment:
Nit. Could we also take the opportunity to test `--report-output-path` here?
```suggestion
      rm -rf "${OLDPWD}/build/benchmark-results" "${OLDPWD}/build/benchmark-results" "${OLDPWD}/build/benchmark-results-old"
     
      elastic-package benchmark -v --report-format xml --report-output file --report-output-path "${OLDPWD}/build/benchmark-results-old" --fail-on-missing
      elastic-package benchmark -v --report-format json --report-output file --fail-on-missing
      elastic-package report --fail-on-missing benchmark --new ${OLDPWD}/build/benchmark-results --old ${OLDPWD}/build/benchmark-results-old --threshold 1
```

Reply:
Umm, tests still don't use `--report-output-path`? :thinking: